pkg/volume/csi/csi_mounter.go

```diff
@@ -247,8 +247,8 @@ func (c *csiMountMgr) podAttributes() (map[string]string, error) {
 		return nil, err
 	}
 
-	// if PodInfoOnMountVersion is not set or not v1 we do not set pod attributes
-	if csiDriver.Spec.PodInfoOnMountVersion == nil || *csiDriver.Spec.PodInfoOnMountVersion != currentPodInfoMountVersion {
+	// if PodInfoOnMount is not set or not v1 we do not set pod attributes
+	if csiDriver.Spec.PodInfoOnMount == nil || *csiDriver.Spec.PodInfoOnMount != currentPodInfoMount {
```

Comment:
Can we get rid of currentPodInfoMount and just check against false?

Reply:
OK